app/views/contacting_creator_mailer/product_level_refund_policies_reverted.html.erb

```diff
@@ -5,12 +5,12 @@
   </p>
   <p>
     If you previously set an account-level refund policy, please review your products to ensure the desired policies are applied.
-  </p>
   <p>
     Soon, we'll be able to process refund requests on your behalf.
   </p>
   <p>
-    You can review your current refund policies or add new ones by editing a product on the  <%= link_to("Products", products_url) %> page.
+    You can review your current refund policies or add new ones by editing a product on the  <%= link_to("Products", products_url) %> page.<br>
+    For more information, see <%= link_to("Set a custom refund policy", "https://gumroad.com/help/article/335-custom-refund-policy.html") %> help article.
```

Comment:
Added the link to the email. Help articles are within the `public/` folder and don't use the routing system.